Bit-Manipulation/test/CountSetBits.test.js

```diff
@@ -0,0 +1,15 @@
+import { CountSetBits } from '../CountSetBits'
+
+test('CountSetBits', () => {
+  const tests = {
+    // binary representation: set bits
```

Comment:
The algorithm accepts decimal representation and counts the bits in its binary representation.  The comment binary representation:set bits in not correct and I will remove it in the next commit.

Reply:
I have changed the comment it should be a decimal representation to set bits. Counting the number of set bits in a binary representation is only counting the number of ones and the algorithm does not do that. It counts bit sets in the decimal representation of the number. The tests and algorithm deal with numbers only and not strings.

Reply:
https://iq.opengenus.org/brian-kernighan-algorithm/
The algorithm accepts an integer as given in the algorithm.

Reply:
> It counts bit sets in the decimal representation of the number.

A decimal representation does not have a notion of bits. A decimal representation has digits from 0 - 9.

> The tests and algorithm deal with numbers only and not strings.

I'm aware. But using numbers as keys for a JS object will coerce them to strings. When passing the numbers into `CountSetBits`, JS will coerce them back to numbers as soon as you start doing arithmetic & bit ops on them.